db_handle/db_crud_operations.py

```diff
@@ -9,25 +9,31 @@ def crud_operations():
 
     def create():
         if user_choice == "customers":
```

Comment:
We can simplify this logic eliminating a significant part of the code below. Due to the fact that currently we have only two tables with pretty much similar columns try to change the algorith as follows:
* Remove the 'if/else' statement
* Modify the variables `customer_id` and `employee_id` names with more generic once, e.g. only `id`
* modify the `POSTGRES_CURSOR` query and change the table name with the variable `user_choice`

In that way when the `user_choice` accepts value `customers` OR `employees`, variable will serve directly as part of the query:
`postgres_conn.POSTGRES_CURSOR.execute(f"INSERT INTO '{user_choice}' values ('{id}', '{first_name}', '{last_name}', '{email_address}', '{phone}')")`

Reply:
Fixed with 25a8cee23c685f5c6a92f07297b4549e2c061fab